client/packages/common/src/intl/locales/en/common.json

```diff
@@ -95,6 +95,7 @@
   "heading.edit-item": "Edit Item",
   "heading.expiring-stock": "Expiring Stock",
   "heading.synchronise-status": "Synchronisation Status",
+  "heading.user-sync": "Sync User",
```

Comment:
:nit I'd prefer a non-abbreviated heading. `Sync User` could also be misunderstood as 'the user for sync'. Does it really make sense to talk about syncing a user? It's more about 'user details' or, more specifically, 'user permissions'.
Maybe `Current User Details`, with `Last successful update`? perhaps not important and 'sync' is ok?

Reply:
`Current User Details` sounds as if we should display user information as well 🤔

Reply:
true! `User Synchronisation`? 
I was wondering if we should specify that it's the currently logged in user that we're talking about here.. just to clarify

Reply:
Update current user? Last successful user update?

Reply:
`User Synchronisation Status`?
The `Update current user` sounds like an action not a status display. I like having `Status` in the heading to be consistent with the `Synchronisation Status` above.

Reply:
[cb50c58](https://github.com/openmsupply/open-msupply/pull/2080/commits/cb50c5802bdfcae5ec3f25d48617ccd05e8d2ed7)